pkg/cmd/browse/browse.go

```diff
@@ -177,6 +177,29 @@ func runBrowse(opts *BrowseOptions) error {
 	}
 	url := ghrepo.GenerateRepoURL(baseRepo, "%s", section)
 
+	wantsWiki := strings.HasSuffix(baseRepo.RepoName(), ".wiki")
+	if wantsWiki {
+		if section == "wiki" {
+			repoName := strings.TrimSuffix(baseRepo.RepoName(), ".wiki")
+			baseRepo = ghrepo.NewWithHost(baseRepo.RepoName(), repoName, baseRepo.RepoHost())
```

Comment:
The following typo causes it to fail as the API call you added is looking for `neovim.wiki/neovim` which does not exist.
```suggestion
			baseRepo = ghrepo.NewWithHost(baseRepo.RepoOwner(), repoName, baseRepo.RepoHost())
```